azurerm/internal/services/datafactory/data_factory_linked_service_azure_databricks_resource.go

```diff
@@ -660,12 +616,12 @@ func constructNumberOfWorkersProperties(minWorkersProperty interface{}, maxWorke
 	return workersConfig, err
 }
 
-func parseNumberOfWorkersProperties(numberOfWorkersProperty string) (interface{}, interface{}, interface{}) {
```

Comment:
We should steer clear of interfaces when we know what we should be returning. `strconv` should catch anything weird happening

Reply:
Agreed, makes sense.